### DISCUSSION TITLE
Add websockets or add an example of how to use websockets with the existing framework.

### DISCUSSION THREAD

**Discussion by linuxguy123** [pyscript/pyscript#1278] [Proposals]
PyScript looks fantastic.   However, I need a websocket for my current project.   Is it possible to use a JS websocket object with PyScript instead of relying on the PyPi websocket ?

Could someone do an example of how to implement a websocket with PyScript ?   Something like the equivalent of this https://pypi.org/project/websockets/

Keep up the great work. 

Edit

https://www.reddit.com/r/PyScript/comments/uvlhsw/pyscript_and_websockets/

https://github.com/HandyGold75/PyScript-WebSocket

**Comment by mtweeman** [comment]
~1 year after OP started this discussion. I see WebSockets reference in docs:
https://docs.pyscript.net/2025.2.2/api/#pyscriptwebsocket

However, when trying to establish connection to existing and working endpoint as shown in the example, WebSocket connection to the endpoint fails.
Is it supported?
